src/main/java/bisq/core/locale/CurrencyUtil.java

```diff
@@ -18,20 +18,12 @@
 package bisq.core.locale;
 
 import bisq.common.app.DevEnv;
+import bisq.core.payment.validation.SpecificAltCoinAddressValidator;
+import lombok.extern.slf4j.Slf4j;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Comparator;
-import java.util.Currency;
-import java.util.List;
-import java.util.Locale;
-import java.util.Objects;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
 import java.util.stream.Collectors;
 
-import lombok.extern.slf4j.Slf4j;
-
```

Comment:
You're using an old version of IDEA that doesn't respect the new `codeStyles/Project.xml` file that's checked into this repository, which means you're mis-organizing imports.

Reply:
Hm, it respected code style from exchange. Is this one different?